src/SpectatorView.Unity/Assets/SpectatorView/Scripts/UI/INetworkConfigurationVisual.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    public delegate void NetworkConfigurationUpdatedHandler(object sender, string ipAddress);
```

Comment:
Can we use action here? And also, can we specify the type for sender instead of object?